README.md

```diff
@@ -86,7 +86,7 @@ The license properties (rules) are stored as a bulleted list within the licenses
 
 #### Conditions
 
-* `include-copyright` - Include a copy of the license and copyright notice with the code.
```

Comment:
@b11094 it looks like your comments are translations. We don't currently have a mechanism for handling translations, but please watch the issue for that (#68). Thanks!